### PR TITLE
mediaconch 16.07

### PR DESCRIPTION
**Pull request by g-maxime**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/mediaconch.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ class Mediaconch < Formula
   depends_on "sqlite"
   # fails to build against Leopard's older libcurl
   depends_on "curl" if MacOS.version < :snow_leopard
+  depends_on "libxslt" unless OS.mac?
```
Comment (DomT4) [inline Formula/mediaconch.rb]: We don't take Linux-related changes to formulae here. Should be removed & forwarded onto Linuxbrew.

**Comment by g-maxime** [comment]
Thank you, I amended my PR

File: Formula/mediaconch.rb (hunk 1 above)
Comment (g-maxime) [inline Formula/mediaconch.rb]: Thank you, I amended my PR

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**